src/index.js

```diff
@@ -1,7 +1,178 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+import React from "react";
+import ReactDOM from "react-dom";
+import Map from "./Map";
+import leafletPip from "leaflet-pip";
+import L from "leaflet";
+import borderData from "./border";
+import counties from "./counties";
 
-ReactDOM.render(
-  <h1>Hello World!</h1>,
-  document.getElementById('root')
-)
\ No newline at end of file
+class App extends React.Component {
+	constructor() {
+		super();
+		this.state = {
+			
+			startButton: false,
+			guessDisabled: true,
+			quitDisabled: true,
+			startingPosition: { lat: 44.4759, lng: -73.2121 },
+			markerPosition: { lat: 44.4759, lng: -73.2121 },
+			markerReveal: {lat: null,lng: null},
+			countyReveal: null,
+			polygon: L.geoJSON(borderData),
+			counties: L.geoJSON(counties)
+		};
+		this.moveMarkerNorth = this.moveMarkerNorth.bind(this);
+		this.moveMarkerSouth = this.moveMarkerSouth.bind(this);
+		this.moveMarkerEast = this.moveMarkerEast.bind(this);
+		this.moveMarkerWest = this.moveMarkerWest.bind(this);
+		this.startButton = this.startButton.bind(this);
+		this.guessButton = this.guessButton.bind(this);
+		this.quitButton = this.quitButton.bind(this);
+		this.checkWithin = this.checkWithin.bind(this);
```

Comment:
If you define these methods as arrow functions (`checkWithin = () => {...}` down on line 34) then you won't need to rebind `this` since arrow functions automatically bind `this` to the containing scope, in this case the class definition.